examples/ota-requestor-app/linux/README.md

```diff
@@ -297,6 +300,3 @@ out/chip-tool otasoftwareupdaterequestor read default-ota-providers 0x858 0 --co
 ```
 
 ## Limitations
-
--   Stores the downloaded file in the directory this reference app is launched
```

Comment:
🎊

Reply:
Should we just remove the Limitations section altogether?

Reply:
I didn't want to seem too presumptuous ;-)

Reply:
I can remove this in the next PR, so as to not re-run the entire CI process for this change.